apps/govquests-api/rails_app/db/migrate/20250115203038_make_badgeable_optional_in_badges.rb

```diff
@@ -1,11 +0,0 @@
-class MakeBadgeableOptionalInBadges < ActiveRecord::Migration[8.1]
-  def change
-    change_column_null :badges, :badgeable_type, true
-    change_column_null :badges, :badgeable_id, true
-
-    remove_index :badges, [:badgeable_type, :badgeable_id]
-    add_index :badges, [:badgeable_type, :badgeable_id],
-      unique: true,
-      where: "badgeable_type IS NOT NULL AND badgeable_id IS NOT NULL"
-  end
-end
```

Comment:
this migration has already run 😬 you need to undo it manually if you want these changes to be removed: https://guides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations